<?xml version="1.0" encoding="UTF-8"?>
<launch>

  <arg name="params_file"/>
  <arg name="output" default="screen"/>

  <node name="ground_truth_mapping_supervisor"
        pkg="ground_truth_mapping"
        type="ground_truth_mapping_supervisor"
        output="$(arg output)"
        required="false">
    <rosparam file="$(arg params_file)"/>
  </node>

</launch>